### ISSUE TITLE
Make left and right arrows in lightbox overlay always on

### ISSUE THREAD

**Issue by GlazerGallery** [woocommerce/woocommerce#24625]
Default behavior of WooCommerce lightbox arrows in which they fade out within three seconds when the mouse is not moved has proved confusing to our visitors, so we would like it changed so that they always stay on, regardless of whether the mouse is moved or not. We believe that the default behavior should be changed so that they are always on, but if you are unwilling to do so, we would like to know if there is code that can be added to our Child Theme. We don’t use Storefront, but to make it clear what the problem is, you can see the behavior I am talking about on this page on the Storefront demo:

https://themes.woocommerce.com/storefront/product/build-your-dslr/#camera-body

To understand what we want to change follow these steps:

1. Click the magnifier icon on the main image.
2. The lightbox overlay opens with the main image and caption, and left and right arrows on either side.
3. Move the mouse into the black area (or off the window). Count slowly to three: “One-one-thousand-two-one-thousand-three…” The arrows disappear.

True, they reappear as soon as the mouse is moved. But we have found that some people find this confusing when they disappear and don’t realize what to do. Worse yet, they haven't noticed the arrows and don't realize that they can advance the images within the lightbox. It may seem very obvious to tech-savvy folks what to do, but apparently it is not so for some people; this is what we are hearing — even one of our own longtime employees got confused by this a couple of months ago! We’ve gotten user complaints about it.

I made a screen recording shared here as a ZIP attachment. It's a 6 MB QuickTime movie.
[WooCommerceArrows.zip](https://github.com/woocommerce/woocommerce/files/3607158/WooCommerceArrows.zip)

In summary, we would like to keep the arrows on all the time, and never fade out until the lightbox is closed by the user. Can you please help? I have asked on a few forums and nobody has offered a workaround.

**Comment by vedanshujain** [comment]
I am not sure if this is the ideal behavior but maybe we can reconsider if there more people think the same. Unfortunately, this is a bug reporting forum and not support forum so we won't be able to help you further, but I think you can add following CSS to your site as a workaround:
```
.pswp__ui--idle .pswp__button.pswp__button--arrow--right {
  opacity: 0.5;
}
.pswp__ui--idle .pswp__button.pswp__button--arrow--left {
  opacity: 0.5;
}
```

**Comment by GlazerGallery** [comment]
Hi, thanks but no, that doesn't do anything. It seems to be a dynamic behavior that is timed somehow. Is it possible that it is governed by a PHP function?

**Comment by GlazerGallery** [comment]
I spoke too soon -- it started working 48 hours later. Not sure why the lag. But thanks for sending it!
